### PR TITLE
Fixed the Bug in team page contact-us section (Email validation is not working)

### LINKED ISSUE #115: Email Validation
In this it is a procedure to check if an email address is deliverable and valid. It catches typos and It also confirms if a particular email address exists with a reliable domain such as Gmail or Yahoo.

@Shruti3004  Please assign this issue to me.

### PR DESCRIPTION
**Pull request by Krishan-2000**
Issue resolved #115  

Bug in Team page contact -us section email validation is not working.

### CODE REVIEW

**Comment by Krishan-2000** [comment]
@ravikr126  Sir, I raised my PR can you please review it.

**Comment by Krishan-2000** [comment]
@harikesh409 I have raised the pr 5 days ago please review it.

File: team.html [hunk 1]
```diff
@@ -43,7 +43,7 @@
                     </li>
 
                     <li class="nav-item">
-                        <a class="nav-link  font-weight-bold nav-items" href="#team">Team</a>
+                        <a class="nav-link  font-weight-bold nav-items" href="team.html">Team</a>
```
Comment (harikesh409) [inline team.html:46]: `#team` is correct only as we are already on the team.html page itself.

**Review by harikesh409 — CHANGES_REQUESTED** [review]

**Comment by Krishan-2000** [comment]
@harikesh409 Sir I have Updated team.html.

**Review by harikesh409 — APPROVED** [review]

**Pull request merged**